server/models/interview.js

```diff
@@ -0,0 +1,37 @@
+let mongoose = require('mongoose');
+
+let Interview = new mongoose.Schema(
+	{
+		owner: {
+			type: mongoose.ObjectId,
+			required: true,
+		},
```

Comment:
For a reference to another model, you have to [name the model in the ref property](https://mongoosejs.com/docs/populate.html)

Reply:
References to owner and guest id have been added.